qCC/ccCommandLineCommands.cpp

```diff
@@ -351,6 +352,45 @@ bool CommandChangeMeshOutputFormat::process(ccCommandLineInterface &cmd)
 	return true;
 }
 
+CommandChangeHierarchyOutputFormat::CommandChangeHierarchyOutputFormat()
+	: CommandChangeOutputFormat("Change hierarchy output format", COMMAND_HIERARCHY_EXPORT_FORMAT)
```

Comment:
I don't know if we currently translate command line stuff, but maybe wrap the text in `tr()` (or `QObject::tr()` if necessary).

Reply:
Added QObject::tr() throughout